
bugs.launchpad.net/hplip/+bug/2096650

src.fedoraproject.org/rpms/hplip/blob/f42/f/hplip-gcc15-stdc23.patch

--- a/scan/sane/ledmi.h	2025-02-27 13:12:29.000000000 +0100
+++ b/scan/sane/ledmi.h	2025-05-05 12:28:33.445527226 +0200
@@ -162,7 +162,7 @@
 int bb_open(struct ledm_session*);
 int bb_close(struct ledm_session*);
 int bb_get_parameters(struct ledm_session*, SANE_Parameters*, int);
-int bb_is_paper_in_adf();         /* 0 = no paper in adf, 1 = paper in adf, -1 = error */
+int bb_is_paper_in_adf(struct ledm_session *);         /* 0 = no paper in adf, 1 = paper in adf, -1 = error */
 SANE_Status bb_start_scan(struct ledm_session*);
 int bb_get_image_data(struct ledm_session*, int); 
 int bb_end_page(struct ledm_session*, int);
